server/controllers/index.js

```diff
@@ -1,3 +1,4 @@
+import likeArticle from './like.controllers';
```

Comment:
I said you should rename this to `likeController` this is still `likeArticle` @amaechi-chuks